app/Http/Middleware/LogRequests.php

```
@@ -42,6 +43,94 @@ public function handle(Request $request, Closure $next): Response
+
+        // 404 errors
+        if ($response->status() === 404) {
+            Log::channel('say')->error('Page not found: ' . $path);
+            return;
+        }
+
+        // Generic page visits for other important pages
+        if (in_array($path, ['about', 'portfolio', 'testimonials', 'projects'])) {
+            Log::channel('say')->info('User exploring ' . $path . ' page');
+        }
+    }
```

PHPMD (2.15.0)

The method addVocalNavigationFeedback() has a Cyclomatic Complexity of 24. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)